content/200-concepts/100-components/02-prisma-client/054-middleware/100-soft-delete-middleware.mdx

```diff
@@ -310,6 +312,7 @@ Run the following sample to see how middleware affects each query:
 
 ```ts
 import { PrismaClient } from '@prisma/client'
+import { PrismaClientKnownRequestError } from '@prisma/client/runtime/library'
```

Comment:
I think this is wrong and should import from the package directly.

Reply:
> I think this is wrong and should import from the package directly.

Absolutely agree. I fix code to directly import that error.
Thank you.

Reply:
Out of interest: How did you end up with this import? Did your IDE suggest this?

Reply:
> Out of interest: How did you end up with this import? Did your IDE suggest this?

yes. I use visual studio code and it suggested this.
I didn't know there was a better way. Thank you for your review. 😄